Solutions/Authomize/Analytic Rules/Privileged_Machines_Exposed_to_the_Internet.yaml

```diff
@@ -0,0 +1,62 @@
+id: 72891de4-da70-44e4-9984-35fcea98d000
+name: Privileged Machines Exposed to the Internet
+kind: Scheduled
+description: These are AWS Ec2 machines that are exposed to the internet. You can further filter by tags so that you can, for example, find exposed machines that are also "privileged".
+severity: High
+queryFrequency: 5m
+queryPeriod: 30m
+triggerOperator: gt
+triggerThreshold: 0
+tactics:
+- Discovery
+- Impact
+relevantTechniques:
+- T1580
+query: |-
+  Authomize_v2_CL
+  | extend EventID = id_s
+  | extend Policy = policy_name_s
+  | extend Severity = severity_s
+  | extend Description = description_s
+  | extend Recommendation = recommendation_s
+  | extend URL = url_s
+  | extend Tactics = tactics_s
```

Comment:
we can also use extend for all columns in a single line
| extend EventID = id_s, Policy = policy_name_s, Severity = severity_s, Description = description_s, Recommendation = recommendation_s, URL = url_s, Tactics = tactics_s

Reply:
Done.